mypy/test/testpythoneval.py

```diff
@@ -61,7 +61,7 @@ def test_python_evaluation(testcase: DataDrivenTestCase) -> None:
     """
     assert testcase.old_cwd is not None, "test was not properly set up"
     mypy_cmdline = ['--show-traceback']
-    py2 = testcase.name.lower().endswith('python2')
+    py2 = 'python2' in testcase.name.lower()
```

Comment:
I think this should be restored to exactly what it was before. I'll see if I can do the edit online myself.